.changeset/pre.json

```diff
@@ -1,11 +1,13 @@
 {
-  "mode": "pre",
+  "mode": "exit",
   "tag": "beta",
```

Comment:
I'm on mobile, but I can't remember - do we still need this?

Reply:
Not sure how this will actually work, been following https://github.com/changesets/changesets/blob/main/docs/prereleases.md

Maybe this will get changed later with the release PR of the changeset bot!? 🤔

Reply:
Yeah! Let's find out! Can always adjust if it's not as we expect